README.md

```diff
@@ -1,174 +1,174 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up a 'Hello Microverse' project <a name="about-project"></a>
+
+**'Hello Microverse' project** is a demo or simple project to get basic understanding of linters and also github flow methods.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">index.html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">style.css</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> The project consits of
+
+- **html**
+- **css**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ npm install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Fentahunn/Hello-microverse.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Hello-microverse
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  manipulate it according to your need.
+```
+
+### Run tests
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Fentahun**
+
+- GitHub: [@githubFentahun](https://github.com/Fentahunn)
+- Twitter: [@twitterFentahun](https://twitter.com/FentahunAgmas)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **simple project**
+- [ ] **It has HTML**
+- [ ] **It has css**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project take it as resource for learning
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding prartner,.They are really supportive.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **How it has been done**
+
+- **what are the key features**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>

```

Comment:
- [ ] [OPTIONAL] `FAQ` ia an `OPTIONAL` part of `README` however if you have decided to include it in the `README` I would suggest you put the `FAQ heading` here as well ` (## ❓FAQ)`